spider.yaml

```diff
@@ -27,7 +27,7 @@ use_references_list: true
 #   file_name: my_fonts
 
 # Location where all the generated references will be stored
-package: assets/res
+package: assets
 
```

Comment:
Lets make general `lib/generated` folder. Here we will store assets generation, localization generation and etc. Assets generation files can be stored in `lib/generated/assets`.